### PR TITLE
[Pavlo Turchyn]Fetch from server

### PR DESCRIPTION
**Pull request by FabianMoretti**

### CODE REVIEW

File: css-practice/task_3/serverUpdate.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    </div>
+  </div>
+</div>`;
+  ourBlog.appendChild(blog);
+  }
+}
+function createFooterBlog(data) {
+  const latestLength = data.latest.length;
+  const blogsLength = data.blogs.length;
+  for (let i = latestLength - 1; i < blogsLength; i++) {
```
Comment (taras1313) [inline css-practice/task_3/serverUpdate.js]: also the problem is, that we have this for loop in both functions. So i would create separate function that makes whole common logic. Like blogRenderer(templateToRender, where to render). If you have some questions to this comment, call me.

File: css-practice/task_3/serverUpdate.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+fetch(blogs)
+  .then((res) => res.json())
+  .then(createBlog)
+
+fetch(blogs)
+  .then((res) => res.json())
+  .then(createFooterBlog)
+
+function createBlog(data) {
+  console.log(data);
```
[the remark this answers is not in the document]
Comment (FabianMoretti) [reply css-practice/task_3/serverUpdate.js]: ready

File: css-practice/task_3/serverUpdate.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+fetch(blogs)
+  .then((res) => res.json())
+  .then(createBlog)
+
+fetch(blogs)
+  .then((res) => res.json())
+  .then(createFooterBlog)
+
+function createBlog(data) {
```
[the remark this answers is not in the document]
Comment (FabianMoretti) [reply css-practice/task_3/serverUpdate.js]: ready

File: css-practice/task_3/serverUpdate.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        <img src="./img/blog-comment.png" alt="seo">
+      </div>
+      <div class="comment-count">${comments}</div>
+    </div>
+  </div>
+</div>`;
+  ourBlog.appendChild(blog);
+  }
+}
+function createFooterBlog(data) {
```
[the remark this answers is not in the document]
Comment (FabianMoretti) [reply css-practice/task_3/serverUpdate.js]: ready

**Comment by taras1313** [comment]
good job

**Pull request merged**